common/ethnicities/wc_ethnicities_vrykul.txt

```diff
@@ -0,0 +1,452 @@
+	}
+	gene_eye_shut = {
+        1 = { name = eye_shut_pos   range = { 0.3 0.5  }    }
+	}
+	gene_bs_eye_corner_depth = {
+        1 = { name = eye_corner_depth_neg    range = { 0 1 }    }
+		1 = { name = eye_corner_depth_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_fold_shape = {
+        1 = { name = eye_fold_shape_pos    range = { 0 1 }    }
+		1 = { name = eye_fold_shape_neg    range = { 0 1 }    }
+	}
+	gene_bs_eye_size = {
+        1 = { name = eye_size_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_upper_lid_size = {
+        1 = { name = eye_upper_lid_size_pos    range = { 0 1 }    }
+	}
+	face_detail_eye_lower_lid_def = {
+        1 = { name = eye_lower_lid_def    range = { 0 1 }    }
+	}
+	face_detail_eye_socket = {
+        1 = { name = eye_socket_01    range = { 0 1 }    }
+		1 = { name = eye_socket_02    range = { 0 1 }    }
+		1 = { name = eye_socket_03    range = { 0 1 }    }
+	}
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_triangle_half	range = { 1 1 } }
+	}
+	
+	###Thicc Eyebrows
+	gene_eyebrows_shape = {
+		1 = { name = avg_spacing_avg_thickness range = { 0.5 1 } }
+		1 = { name = avg_spacing_high_thickness range = { 0.5 1 } }
+		1 = { name = far_spacing_avg_thickness range = { 0.5 1 } }
+		1 = { name = far_spacing_high_thickness range = { 0.5 1 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = vrykul_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = vrykul_beards 		range = { 0.0 1.0 } }
+	}
+	special_eyebrows = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.7 0.9 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
+	expression_forehead_wrinkles = {
+        1 = {  name = forehead_wrinkles_01             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_02             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_03             range = { 0.1 1 }     }
+	}
+	complexion = {
+        1 = {  name = complexion_1             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_3             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_4             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_5             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_6             range = { 0.43 0.43 }     }
+	}
```

Comment:
Why no sexy lips?
👄

Reply:
> 
> 
> Why no sexy lips?
> 👄

Added sexy lips in 33beb4e0d698d2a6470310f62843026dd1eb924d.